CMakeLists.txt

```diff
@@ -265,7 +265,7 @@ else (APPLE)
       if (BUILD_64 STREQUAL "ON")
          set(USE_SSE FALSE)
       endif(BUILD_64 STREQUAL "ON")
-   # Set compiler options for VS2017 toolchain.
+   # Set compiler options for VS2017/19 toolchain.
```

Comment:
Should be just VS2019 here, I think, since you've removed all other references to VS2017.

Reply:
VS2017 still works though.

Reply:
The options following that comment are valid for the VS2017 and VS2019 toolchain

Reply:
I originally misunderstood the purpose of this PR. I thought it was to drop VS2017 entirely, when actually it's to make VS2019 the default build system but still allow VS2017.

So I have no complaints about this comment anymore.

Reply:
yes, the intention was to switch to VS2019, while not disallowing VS2017